### ISSUE TITLE
Any benchmark result?

### ISSUE THREAD

**Issue by soloice** [XMUNLP/XMUNMT#3]
Hi,
I'm wondering if this reproduce could achieve similar performance with the original RNNSearch implementation (GroundHog). Is there any benchmark result?

**Comment by Playinf** [comment]
Hi, thank you for your comments. I've updated a benchmark on WMT17's De-En dataset. The BLEU (case insensitive) score is calculated using mteval.perl. More benchmarks will be updated soon.

**Comment by soloice** [comment]
It's very kind of you to post this. A BLEU of 30.42 in DE-EN translation is impressive. Looking forward to more benchmarks.
Btw, is this repo just a simple reproduction of RNNSearch in TensorFlow? Or does it incorporate some new features/algorithms/etc?

If it is the former case, can original RNNSearch implementation [(GroundHog)](https://github.com/lisa-groundhog/GroundHog) achieve similar results with the same hyper-parameter/optimizer/preprocessing(i.e.: BPE, etc)? If latter, is there a document/paper describe the difference between RNNSearch and this repo?

Hope I've made myself clear. Many thanks~

**Comment by Playinf** [comment]
The architecture of RNNsearch is identical to GroundHog. I believe it is possible for GroundHog to achieve similar results. The differences are listed below:
1. We use Adam instead of Adadelta as the optimizer
2. The parameters are initialized uniformly between [-0.08, 0.08]. The GroundHog uses orthogonal initialization.
3. We also applied dropout which further improves the performance.
4. The GroundHog lacks validation functionality and the early stopping technique. We implement both in our codes.

**Comment by soloice** [comment]
As you illustrated above, I guess I could use this reproduction as the RNNSearch baseline in my research. (The original implementation is not maintained any more and relies on so many obsolete libraries, thus it's too difficult to set up its environment without affecting other users on our server = =)
